src/pages/login/loginContainer.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect } from "react";
+import { useFormik } from "formik";
+import { useLocation, Redirect } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+
+import Login from "./login";
+
+import AUTHFORMDATA from "config/constants/authformdata";
+import { userActions } from "config/actions/userActions";
+
+const validate = (values) => {
+  const errors = {};
+  if (!values.email) {
+    errors.email = "Required";
+  } /*else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)) {
+    errors.email = "Invalid email address";
+  }*/
+  if (!values.password) {
+    errors.password = "Required";
+  }
+  return errors;
+};
+
+const LoginContainer = () => {
+  const user = useSelector((state) => {
+    return state.authenticationReducer.user;
+  });
```

Comment:
Why do you use return here? And move please all state selections to a separate file.
```suggestion
  const user = useSelector((state) => state.authenticationReducer.user);
```